SingularityUI/app/collections/TaskSearchResults.coffee

```diff
@@ -0,0 +1,21 @@
+Collection = require './collection'
+
+TaskHistoryItem = require '../models/TaskHistoryItem'
+
+class TaskSearchResults extends Collection
+
+    model: TaskHistoryItem
+
+    getQueryParams: ->
+        return $.param(_.pick(@params, _.identity))
```

Comment:
We wanted to exclude null items, things like deployId=&requestId=somevalue were messing with the optional query params on the backend, making them optionals of empty strings instead of optional absents